### PR TITLE
chore(DataList): added additional rtl styling

### LINKED ISSUE #5821: Implement RTL: simple list, data list, description list

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #5821

### CODE REVIEW

**Review by mcoker — APPROVED** [review]
😍

**Comment by thatblindgeye** [comment]
@mcoker added the comment regarding the notification badge update for the next breaking change

File: src/patternfly/components/NotificationBadge/notification-badge.hbs [hunk 1]
```diff
@@ -14,6 +14,9 @@
     {{#if notification-badge--attribute}}
       {{{notification-badge--attribute}}}
     {{/if}}>
+    {{!-- TODO in Breaking Change: Add a wrapper element around the icon to follow
+          how other components' markup is structured for icons
+     --}}
```
Comment (mcoker) [inline src/patternfly/components/NotificationBadge/notification-badge.hbs:19]: 👍

**Review by srambach — APPROVED** [review]
LGTM!

**Pull request merged**